### PR TITLE
docsp-34251 - c++ version updates

### PR DESCRIPTION
**Pull request by mongoKart**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-ecosystem/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-34251
Staging - https://preview-mongodbmongokart.gatsbyjs.io/drivers/docsp-34251-version-update/cxx/

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [ ] Are all the links working?
- [ ] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/includes/mongodb-compatibility-table-cxx.rst [hunk 1]
```diff
@@ -18,6 +18,19 @@
      - MongoDB 3.0
      - MongoDB 2.6
      - MongoDB 2.4
+   * - 3.9
```
Comment (rustagir) [inline source/includes/mongodb-compatibility-table-cxx.rst:21]: S: just for neatness, add a newline between entries

File: source/includes/language-compatibility-table-cxx.rst [hunk 2]
```diff
@@ -10,6 +10,13 @@
      - C++11
      - C++03
 
+   * - 3.9.0
```
Comment (rustagir) [inline source/includes/language-compatibility-table-cxx.rst:13]: S: instead of adding a new entry, what if you changed the previous entry to be `>3.8`

**Review by rustagir — APPROVED** [review]
lgtm with two nit suggestions, also maybe while you're working on this page add the metatags!

File: source/includes/language-compatibility-table-cxx.rst (hunk 2 above)
Comment (mongoKart) [reply source/includes/language-compatibility-table-cxx.rst:13]: love it

**Pull request merged**